### ISSUE TITLE
pre-training

### ISSUE THREAD

**Issue by CQIITLAB** [amirbar/DETReg#24]
I used 3424 images from Imagenet100 for pre-training. And then fine-tuning on the COCO data set yields almost the same accuracy. Why?

**Comment by amirbar** [comment]
It sounds like you you are missing some images in the pretraining. IN100 should have around ~100k images.
